dev_guide/application_health.adoc

```diff
@@ -44,6 +44,21 @@ Set a readiness check by configuring the
 
 |===
 
+The exact timing of a probe is controlled by two fields,
+both expressed in units of seconds:
+
+|====
+| Field | Description
+
+| `*initialDelaySeconds*`
+| How long to wait after the container starts to begin the probe.
+
+| `*timeoutSeconds*`
+| How long to wait for the probe to finish (default: `1`).
+If this time is exceeded, OpenShift considers the probe to have failed.
```

Comment:
@nak3
Hmmm, i have a doubt: Does OpenShift use that version of Kubernetes?  I just refreshed the repo from https://github.com/openshift/origin and see:

    $ git grep initialDelaySeconds | wc -l
    11
    $ git grep timeoutSeconds | wc -l
    34
    $ git grep successThreshold | wc -l
    0
    $ git grep failureThreshold | wc -l
    0

My understanding is that OpenShift embeds a version of Kubernetes, not necessarily the bleeding edge (from upstream), and that this process is called "vendoring".  What am i missing?